### ISSUE TITLE
remove textBubble

### ISSUE THREAD

**Issue by cainoeraabele** [badoo/Chatto#313]
hi i have a question
is it possible remove textBubble ??

tx cainoeraabele

**Comment by charvoa** [comment]
Yes, in order to do that you should just fork the project and modify the appearence of the bubble.
